decompiler/structures/ast/switch_node_handler.py

```diff
@@ -181,16 +181,21 @@ def __check_for_zero_case_condition(self, condition: Condition) -> Optional[Tupl
                 continue
             if ssa_condition is None:
                 ssa_condition = self.__get_z3_condition(ExpressionUsages(condition, tuple_ssa_usages))
+                if ssa_condition is None:
+                    continue
```

Comment:
If we can not compute the `ssa_condition`, then it will always be `None`, so we can directly return `None` at this place
```suggestion
                if (ssa_condition := self.__get_z3_condition(ExpressionUsages(condition, tuple_ssa_usages))) is None:
                    return None
```

Reply:
done